README.md

```diff
@@ -3,32 +3,34 @@ Export any Node.js `fs`-like object as a FUSE filesystem
 
 ## Node.js `fs` methods
 
-The next `fs` methods are being used:
+The following `fs` methods directly map to FUSE operations:
```

Comment:
They are direct map in their behaviour, but not on their signature. Maybe this can lead to confusion, can you reword it? :-)